q2_types/per_sample_sequences/_format.py

```diff
@@ -46,6 +47,47 @@ def sniff(self):
         return sniffer(str(self))[0]
 
 
+class QIIME1FastqManifestFormat(model.TextFileFormat):
+    def sniff(self):
+        with self.open() as fh:
+            header = fh.readline()
+            return header.strip() == \
+                'filename,read_number,lane_number,phred_offset'
+
+
+class QIIME1DemultiplexedFastqFormat(model.TextFileFormat):
+    def sniff(self):
+        fastq_sniffer = skbio.io.io_registry.get_sniffer('fastq')
+        if fastq_sniffer(str(self))[0]:
+            generator = skbio.io.read(str(self), constructor=skbio.DNA,
+                                      format='fastq', verify=False,
+                                      compression='gzip',
+                                      phred_offset=33)
+            try:
+                for seq, _ in zip(generator, range(5)):
+                    if len(seq.metadata['id'].split('_')) == 1:
```

Comment:
Use `.split('_', maxsplit=1)`, check that you receive a list of exactly two elements, and check that both pieces are not empty